plugins/main/public/components/endpoints-summary/dashboard/components/outdated-agents-card.scss

```diff
@@ -0,0 +1,26 @@
+.wazuh-outdated-agents-panel {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: auto;
+  flex-direction: column;
+  cursor: pointer;
+}
+
+.wazuh-outdated-metric .euiTitle {
+  font-size: 4.5rem;
+  line-height: inherit;
+}
+
+.wazuh-outdated-metric small {
+  font-size: 1.5rem;
+}
+
+.wazuh-outdated-icon {
+  width: 3.5rem;
+  height: 3.5rem;
+}
+
+div.euiPopover__panel.euiPopover__panel-isOpen.euiPopover__panel--bottom {
+  scrollbar-width: none;
+}
```

Comment:
**nitpick**: this could be affecting global styles.

Reply:
## Fixed
CSS Style is added using the panelStyle property of the EuiPopover component so that it does not affect global styles